models/andeson_2007.cpp

```diff
@@ -23,8 +23,8 @@ ncmt=1, trans=11
 D_CENT = 1; 
 double NORM_WT = 70;
 double NORM_PMA = 34.8;
-double CLi = 3.83*pow(wt/NORM_WT,0.75)*(pow(pma, 3.68)/(pow(pma, 3.68) + pow(33.3, 3.68)))*(516*pow(e, (0.00823*((PMA-40)/52-40)))/scr)/6)*Fventilation;
-double Vi = 39.4*(wt/NORM_WT)*Finotrope;
+double CLi = 3.83*pow(wt/NORM_WT,0.75)*(pow(pma, 3.68)/(pow(pma, 3.68) + pow(33.3, 3.68)))*(516*pow(e, (0.00823*((PMA-40)/52-40)))/scr)/6)*Fvent;
```

Comment:
so these are both incorrect. Think of what you are setting as of now. You are saying multiply by 0, so both CL/V will be 0.

In reality, similar to the lo paper, these are likely placeholders for more complex equations like `(1.5*on_vent)`. The `Fvent` stands, essentially, for fraction corresponding to vent status. We need to actually go through the original anderson paper to find the actual values